### ISSUE TITLE
Stream randomly stop

### ISSUE THREAD

**Issue by woowoots** [alamminsalo/orion#42]
Hello, since somes days, somes stream randomly "freeze" ( like if it was dropping frame ) and just stop

It's an orion issue, since if i watch at the same time the stream on twitch, there is no issues at all.

**Comment by alamminsalo** [comment]
Could this be related to [#43](https://github.com/alamminsalo/orion/issues/43)?

**Comment by woowoots** [comment]
Yes, in the ned thoses "issues" where "merged" in the discussion, guess that one can be close / deleted

**Comment by ahjolinna** [comment]
I notice this problem on my old ultrabook (Intel Core i3-3217U/ivybridge mobile), using [GeckoLinux Plasma "NEXT" (openSUSE Leap 42.2 base w/ plasma 5.9 and Qt5.8)](https://groups.google.com/forum/#!topic/geckolinux-updates/mG-f61jc5A4)....only difference is that whole orion freezes for a while with VODs and with live streams playback stops about every 30min.

orion strace output(138.6MiB):
gdb didnt say anything

-------

@alamminsalo : have you thought about adding native logger module support for systemd journal?
if you wonder why it would be useful?

- single place to look for log output
- no need to handle log file creation/rotation/collisions/running out of space
     - just send it to the Journal interface and let it handle the rest
- the Journal API supports priorities and other metadata
     - users can see right away which messages are critical/non critical due to the build in journalctl message highlighting
- powerful filtering & cross reference
    - users can filter out just the bomi log output or check what happened at the same time as the bomi message was logged (bomi error accompanied by a x server error, etc.)
- powerful time based lookup - users can check all bomi log messages from last run, last day, last hour, etc.

More Journal resources:
http://www.freedesktop.org/software/systemd/man/journalctl.html
http://0pointer.de/blog/projects/journal-submit.html
